server.js

```diff
@@ -7,11 +7,28 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
+const Thought = mongoose.model('Thought', {
+  message: {
+    type: String,
+    required: [true, 'Message Required'],
+    minLength: [5, 'Message too short, minimum 5 characters'],
+    maxLength: [140, 'Message too long, maximum 140 characters']
```

Comment:
This should be minlength and maxlength without big L. If it is with big L is not working.